doc/design.md

```diff
@@ -273,6 +295,121 @@ Returns:
     * tbd
 
 
+### **`POST`** `/oeci_login` ?
+
+## Question
+
+What's the workflow for accessing OECI?
```

Comment:
Based on what Nick already said, I'm assuming OECI auth looks like this:

- send username/password credentials in a request
- In the response, OECI server sets a cookie which authenticates future requests
- The cookie expires after 5 minutes of inactivity

If this is not how OECI works then my comments below should possibly be disregarded. Additional info on OECI auth if applicable would be helpful.

After reading through the below, I'm not sure I totally understand the proposed designs, but here are my thoughts:

If OECI auth uses a cookie, redirecting the user to the OECI login form/page won't work because the Javascript running in our app (on a different URL/domain) will never be able to read the cookie, and thus unable to make its own requests to OECI or pass that cookie to the backend to do so on its behalf.

AFAIK the only feasible approach is for the user to input their credentials into our React app, have that app send the credentials to the backend, and then have the backend send the credentials to OECI, then get the cookie back if they're correct. 

The backend can then send the cookie back to the frontend to send with future requests (preferred) or store it on the backend (seems cumbersome).

Reply:
Cool. I just posted a new response on this issue to Nick's comment above, here: https://github.com/codeforpdx/recordexpungPDX/pull/361#discussion_r324431898 , to zero in on what works with both the Crawler and our desired frontend workflow.

(This is my attempt to "merge" the comment "branches" on this PR :) )